charts/backstage/backstage-rhtap-values.yaml

```diff
@@ -203,6 +206,14 @@ upstream:
 
     appConfig:
       backend:
+{% if ocp4_workload_redhat_developer_hub_orchestrator_enabled %}
+        auth:
```

Comment:
Moved this here, since the previous placement was incorrect. Backstage having two `auth` blocks is confusing!